code/modules/antagonists/heretic/items/heretic_blades.dm

```diff
@@ -21,6 +21,13 @@
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "rend")
 	var/after_use_message = ""
 
+/obj/item/melee/sickly_blade/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/butchering, \
+	speed = 3 SECONDS, \
+	effectiveness = 95, \
+	)
```

Comment:
They should already have an effectiveness of 100, this is lowering it, no?
Also you can increase the speed to this by having a tool_speed of 3, so there's no need to be snowflaking this and uses the intended vars for its intended use.

Reply:
I think you're right on the effectiveness issue, having any value less than 100 will cause you to yield less/lose yields. But I believe toolspeed doesn't affect butchering speed. All the other instances of adding the butchering component seem to change the speed var directly.

Reply:
> They should already have an effectiveness of 100, this is lowering it, no? Also you can increase the speed to this by having a tool_speed of 3, so there's no need to be snowflaking this and uses the intended vars for its intended use.

how is this snowflaking its just setting the component to a specific value the creator wants. arbitrarily setting toolspeed just for use in butchering seems silly?

Reply:
I think what he means is not directly adding the component, just giving it a toolspeed so that when the component is added by the blade being sharp, it is effectively doing the same job.

I do think they should have a tool behaviour like TOOL_KNIFE if that is the approach Willard wants, though.

Reply:
As appears and if understood the butchering code correctly, the only thing that changes with effectiveness is the amount of meat ._.

Reply:
> arbitrarily setting toolspeed just for use in butchering seems silly?

The tool speed is how efficient it's supposed to be, you're supposed to change that if you want speed changed.